x/nft/types/nft.go

```diff
@@ -14,13 +14,13 @@ import (
 type NFT interface {
 	GetID() uint64
 	GetOwner() sdk.AccAddress
-	SetOwner(address sdk.AccAddress)
+	SetOwner(address sdk.AccAddress) BaseNFT
```

Comment:
this should return an `NFT`, not a `BaseNFT`

Reply:
Agree, I was just looking at the handler input type and I just returned the same type, just fixed it